images/nginx-drupal/drupal.conf

```diff
@@ -90,34 +101,39 @@ server {
     deny all;
     access_log off;
     log_not_found off;
+    return 404;
```

Comment:
why do we need a specific 404 here? the `deny all` already returns with an `403`? see http://nginx.master.drupal-example-9.test1.amazee.io/patches/asdf